MathLib/Point3d.h

```diff
@@ -50,6 +50,14 @@ double sqrDist(MathLib::Point3d const& p0, MathLib::Point3d const& p1)
 	return MathLib::scalarProduct<double,3>(v,v);
 }
 
+/// Computes the squared distance between the orthogonal projection of the two
+/// points \c p0 and \c p1 onto the \f$xy\f$-plane.
+inline
+double sqrDist2d(MathLib::Point3d const& p0, MathLib::Point3d const& p1)
+{
+	return std::pow(p0[0]-p1[0], 2) + std::pow(p0[1]-p1[1], 2);
```

Comment:
The are overloads of `std::pow` for integer exponents that are nearly as fast as the multiplication. I changed the code according your suggestion.